lib/SyTest/Federation/Server.pm

```diff
@@ -490,8 +490,6 @@ sub on_request_federation_v1_send
       }
 
       next if $self->on_event( $event );
-
-      warn "TODO: Unhandled incoming event of type '$event->{type}'";
```

Comment:
Just checking this is intentional

Reply:
ah, yes it is. The updated code spits out more of these and I don't think we really need the warning.

Reply:
Fair, I've wondered about removing this myself!